src/natcap/invest/carbon.py

```diff
@@ -94,16 +94,16 @@
                 "A table that maps the land-cover IDs to carbon pools.  "
                 "The table must contain columns of 'LULC', 'C_above', "
                 "'C_Below', 'C_Soil', 'C_Dead' as described in the User's "
-                "Guide.  The values in LULC must at least include the LULC "
-                "IDs in the land cover maps."),
+                "Guide.  The LULC column must include all the pixel values "

```

Comment:
removing extra space char
```suggestion
                "Guide. The LULC column must include all the pixel values "

```

Reply:
another thing for our future style guide! I've seen the double spaces around in some models and it may be an artifact of [some older typing style standards](https://www.instructionalsolutions.com/blog/one-space-vs-two-after-period). I agree it should be taken out.

Reply:
I believe 'lucode' is correct here because it's referring to the column header, which must be 'lucode' when converted to lower-case. LULC isn't wrong, but 'lucode' may be more specific?

Reply:
I agree again, but if you look at the example in the [UG](http://releases.naturalcapitalproject.org/invest-userguide/latest/carbonstorage.html#data-needs) and the sample data, there is an optional col in the Carbon Pools table labeled 'LULC_name', so referring to "The LULC column..." could be confusing to users. Yes, I am aiming for more precise language.

Reply:
Oh, good point about the optional column! Yea, I think we could take out "optional" columns from the UG/sample data entirely, since they're functionally no different from any other extra column that might be in a table

Reply:
While I agree that taking out the columns that are not required would be a nice, minimal set of sample data, the "LULC name" column and the like does make biophysical tables more readable, which could help a user infer meaning from the table itself.  So I know it's a bonus column and not strictly necessary, but I'd push to retain columns like this that add helpful context.

On the other hand, if our sample data have biophysical coefficients that are no longer used, I'd be inclined to remove those!

Reply:
I agree that having certain optional cols in the UG and sample data improve readability and can be helpful to users, but I think it's important that we are careful they don't contribute to any confusion.